zeebe/stream-platform/src/main/java/io/camunda/zeebe/stream/api/ProcessingResultBuilder.java

```diff
@@ -84,6 +84,19 @@ ProcessingResultBuilder withResponse(
    */
   ProcessingResultBuilder resetPostCommitTasks();
 
+  /**
+   * Signals to the streaming platform to process the result in a separate command batch.
+   *
+   * <p>This is useful for cases where the processing of the records is complex or requires
+   * additional resources, and you want to ensure that they are not mixed with other records in the
```

Comment:
:thought_balloon: In our case, it's also that the various commands are independent of each other, and one failing shouldn't have an impact on the other.

Reply:
That goes for the other similar comments.